cmake/configure/configure_20_trilinos.cmake

```diff
@@ -284,37 +284,37 @@ macro(feature_trilinos_find_external var)
             STATUS
             "  Tpetra wasn't configured with support for 64-bit global indices\n"
             "  but deal.II is configured to use 64-bit global indices.\n"
-            "  Either reconfigure deal.II with -DDEAL_II_WITH_64BIT_INDICES=OFF.\n"
+            "  Either reconfigure deal.II with -DDEAL_II_WITH_64BIT_INDICES=OFF\n"
             "  or rebuild Trilinos with -DTPETRA_INST_INT_LONG_LONG=ON"
           )
           if(DEAL_II_HAVE_TPETRA_INT_LONG)
             message(
               STATUS
-              "  and -DTPETRA_INST_INT_LONG=OFF"
+              "  and -DTPETRA_INST_INT_LONG=OFF."
```

Comment:
If the `if` does not trigger, nor the `elseif` below, then we are left with a sentence that doesn't end in a period in line 288. I decided that's acceptable.